frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -558,24 +558,33 @@ const ManagePolicyPage = ({
 
         return turnedOff || turnedOn || updatedSwId;
       });
+
       if (!changedPolicies.length) {
         renderFlash("success", "No changes detected.");
         return;
       }
+
       const responses: Promise<
         ReturnType<typeof teamPoliciesAPI.update>
       >[] = [];
-      responses.concat(
-        changedPolicies.map((changedPolicy) => {
-          return teamPoliciesAPI.update(changedPolicy.id, {
-            // "software_title_id": null will unset software install for the policy
-            // "software_title_id": X will set the value to the given integer (except 0).
-            software_title_id: changedPolicy.swIdToInstall || null,
-            team_id: teamIdForApi,
-          });
-        })
-      );
+
+      changedPolicies.forEach((changedPolicy) => {
+        responses.push(
+          teamPoliciesAPI
+            .update(changedPolicy.id, {
+              software_title_id: changedPolicy.swIdToInstall || null,
+              team_id: teamIdForApi,
+            })
+            .catch((error) => {
```

Comment:
Did you confirm that one error doesn't prevent the rest of the calls from being cancelled?

Reply:
It'd be nice to avoid combining `try..catch...` and `.catch` syntax if possible, is there a way to do this with just try/catch?

Reply:
Found a _promising_ solution using Promise.allSettled! lmkwyt!